src/Compilers/CSharp/Test/Emit/Attributes/AttributeTests.cs

```diff
@@ -62,6 +62,125 @@ class C
             CompileAndVerify(source, sourceSymbolValidator: attributeValidator, symbolValidator: null);
         }
 
+        [Fact]
+        [WorkItem(20741, "https://github.com/dotnet/roslyn/issues/20741")]
+        public void TestNamedArgumentOnStringParamsArgument()
+        {
+            var source = CreateCompilationWithMscorlib46(@"
```

Comment:
Minor point: Can use `CreateStandardCompilation` in these tests.

Reply:
I needed 46 for those reflection APIs.